scripts/pyenv_setup.sh

```diff
@@ -62,6 +63,7 @@ _append_to_startup_script() {
 }
 
 append_to_config() {
+  [[ ! -f "$1" ]] && touch "$1"
```

Comment:
I've seen EvanP be distracted by the file not being present. I *might* need a better fix. I will move this work to another PR.